windows/add_to_path.tw2

```diff
@@ -9,26 +9,26 @@ To make Exercism usable on your computer, we need to move it to the right folder
 In the Command Prompt window, copy & paste:
 
 ```
-start %LOCALAPPDATA%\Microsoft\WindowsApps
+start "%LOCALAPPDATA%\Microsoft\WindowsApps"
 ```
 
 and hit the 'Enter' key.
 
 ### Verify
 
-A new window, showing a folder, will show up. This is where we need to place the `exercism` file. It will look something like this, but it might not be empty for you.
+A new window, showing a folder, will show up. This is where we need to place the `Exercism` file. It will look something like this, but it might not be empty for you.
 
 ![cmd.exe folder](https://transfer.sh/LwF0S/win10-cmd-folder.png)
 
 
-## Step 2: Moving the `exercism` file
+## Step 2: Moving the `Exercism` file
 ### Exercise
 
-Find the `exercism` file in the folder we opened earlier, when we downloaded Exercism. From this folder, drag the file (or copy & paste it) to the folder we just opened, labeled `WindowsApps`.
+Find the `Exercism` file in the folder we opened earlier, when we downloaded Exercism. From this folder, drag the file (or copy & paste it) to the folder we just opened, labeled `WindowsApps`.
```

Comment:
This should be `exercism` in the first case. Maybe instead of _Exercism_ in the second, we can say _when we downloaded the zip archive_ or something.